src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1459,6 +1465,7 @@ class LineChart extends TemporalXAxisComponent {
                                 .classed("tsi-lineChart", true)
             this.svgSelection = this.targetElement.append("svg")
                                             .attr("class", "tsi-lineChartSVG tsi-chartSVG")
+                                            .attr('title', this.getString('Line chart'))
```

Comment:
it seems like this still doesnt really receive focus in the tab order though?

Reply:
as far as I can tell that's not a requirement/expected, just that the svg has a title

Reply:
sounds good to me